jwt_verify_lib/verify.h

```diff
@@ -21,10 +21,13 @@
 namespace google {
 namespace jwt_verify {
 
+typedef std::vector<std::string> jwt_audience_list;
```

Comment:
We try to following this coding style:  https://google.github.io/styleguide/cppguide.html

Please use CamelCase for Type, Class,  and lower_case for variables.

Reply:
Beside,  we try to avoid typedef. It is NOT easy to read.  You have to search the code to find out the actual type.
For just simple vector, just use std::vector directly.

Reply:
Fixed in latest commit.